public/sass/pages/_explore.scss

```diff
@@ -98,6 +98,10 @@
   padding: 10px 2px;
 }
 
+.explore-toolbar-content-item.timepicker {
```

Comment:
Needed to add this so that the custom ranges in TimePicker appears in front of Explore panels.